tests/cmd-fleet-add-devices-test.toit

```diff
@@ -45,40 +45,41 @@ run-test test-cli/TestCli fleet-dir/string:
     aliases := [alias1, alias2]
     test-cli.run [
       "fleet",
-      "create-identity",
-      "--output-directory", tmp-dir,
+      "add-device",
+      "--format", "identity",
+      "--output", "$tmp-dir/$(id).identity",
       "--name", "test-name",
       "--alias", (aliases.join ","),
-      "$id",
+      "--id", "$id",
     ]
-    test-write-identity test-cli fleet-dir tmp-dir
+    test-extract-identity test-cli fleet-dir tmp-dir
     check-and-remove-identity-files fleet-dir tmp-dir
         --id=id
         --name="test-name"
         --aliases=aliases
 
     test-cli.run --expect-exit-1 --allow-exception [
       "fleet",
-      "create-identity",
-      "--output-directory", tmp-dir,
-      "$id",
+      "add-device",
+      "--output", "$tmp-dir/other.identity",
```

Comment:
No need for format? Or is this supposed to fail with exit code 1 here. Probably.

Reply:
The 'format' was missing.
I added a comment what we are testing.
done.